include/z64save.h

```diff
@@ -1282,16 +1282,21 @@ typedef enum {
 // Also related to moon child
 #define WEEKEVENTREG_84_20 PACK_WEEKEVENTREG_FLAG(84, 0x20)
 
-#define WEEKEVENTREG_84_40 PACK_WEEKEVENTREG_FLAG(84, 0x40)
+#define WEEKEVENTREG_RECEIVED_RED_POTION_FOR_KOUME PACK_WEEKEVENTREG_FLAG(84, 0x40)
 #define WEEKEVENTREG_84_80 PACK_WEEKEVENTREG_FLAG(84, 0x80)
 #define WEEKEVENTREG_85_01 PACK_WEEKEVENTREG_FLAG(85, 0x01)
 
 // Unconfirmed: "Bombers Hide & Seek in Progress"
 #define WEEKEVENTREG_85_02 PACK_WEEKEVENTREG_FLAG(85, 0x02)
 
 #define WEEKEVENTREG_85_04 PACK_WEEKEVENTREG_FLAG(85, 0x04)
-#define WEEKEVENTREG_85_08 PACK_WEEKEVENTREG_FLAG(85, 0x08)
-#define WEEKEVENTREG_85_10 PACK_WEEKEVENTREG_FLAG(85, 0x10)
+
+// These get set per location, if Kotake tries to give the player a red potion to save koume,
```

Comment:
Being crazy pedantic here:
```suggestion
// These get set per location, if Kotake tries to give the player a Red Potion to save Koume,
```